tidb-server/main.go

```diff
@@ -188,6 +191,21 @@ func syncLog() {
 	}
 }
 
+func acquireTempDirLock() {
```

Comment:
Could this function be named as `initializeTempDir` and do initialization and lock mechanism together here?

Reply:
Since we have clean the temp storage path before we first use it, so I make it as a sub-directory of the setting `tmp-storage-path`. PTAL